### ISSUE TITLE
reuse existing MongoDB connection

### ISSUE THREAD

**Issue by mcollina** [fastify/fastify-mongodb#39]
This would be mostly helpful doing tests, as we can speed things up significantly.
Also, https://github.com/mcollina/mongo-clean can help for tests as well.

**Comment by mcollina** [comment]
My bad. This is already supported but not documented.

**Comment by shijin-p** [comment]
I created a mongodb atlas trial account. After connecting to mongodb, I am able to fetch data from db in request. I bench marked this request and got 90 req/sec !
I removed the db query from request, it became 99 req/sec
I removed the mongodb plugin register and got 9000 req/sec !!!
Why is just registering mongdb and not making any db query in request effecting concurrency like this. Can someone give some suggestions!
The code snippet I used to register mongodb is below.
`  await fastify.register(Mongodb, {
     forceClose: true,
     url: process.env.MONGODB_URI
   })`
